wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -730,6 +730,15 @@ public final class SessionManager: NSObject, SessionManagerType {
             if deleteAccount {
                 self?.deleteAccountData(for: account)
             }
+
+            // Clear tmp directory when the user logout from the session.
+            guard let tmpDirectoryPath = URL(string: NSTemporaryDirectory()) else { return }
```

Comment:
The temp directory is not scoped to the current user right? It's just a general temp directory for the whole application?

Reply:
yes it is the temp directory for entire application.